conduit/plugins/importers/syncmode/metrics.go

```diff
@@ -0,0 +1,18 @@
+package syncmode
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Prometheus metric names
+const (
+	GetAlgodRawBlockTimeName = "get_algod_raw_block_time_sec"
+)
+
+// Initialize the prometheus objects
+var (
+	GetAlgodRawBlockTimeSeconds = prometheus.NewSummary(
+		prometheus.SummaryOpts{
+			Subsystem: "indexer_daemon",
```

Comment:
maybe the subsystem/prefix here should be the plugin name. same for algod_importer.

Reply:
Good idea...done.